about.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="build/about_style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>About Summit</title>
+  </head>
+  <body>
+    <div class="about">
+      <div class="header-style">
+        <div class="header-top">
+          <ul class="container">
+            <li>
+              <a href="https://web.facebook.com/Genshinimpact/?_rdc=1&_rdr"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/GenshinImpact?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="https://andrianarivo.github.io/portfolio/">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+        <nav>
+          <a class="hamburger" href="#" id="hamburger"><i class="fa-sharp fa-solid fa-bars fa-2xl"></i></a>
+          <div class="container" id="menu">
+            <a href="./index.html"><img src="images/Genshin_Impact_logo.png" /></a>
+            <a href="#" id="xmark"><i class="fa-solid fa-xmark fa-xl"></i></a>
+            <ul>
+              <li><a href="./about.html" class="menu-item">About</a></li>
+              <li><a href="./index.html#program" class="menu-item">Program</a></li>
+              <li><a href="#" class="menu-item">Join</a></li>
+              <li><a href="#partners" class="menu-item">Sponsor</a></li>
+              <li><a href="#" class="menu-item">News</a></li>
+              <li class="campaign menu-item"><a href="#">Campaign</a></li>
+            </ul>

```

Comment:
- [ ] Kindly apply the hover effect on the nav items. To ensure a pixel-perfect design. 😉
<img width="269" alt="image" src="https://user-images.githubusercontent.com/41163571/233133497-63b2d716-cfd6-4755-ab83-9508c76c3d83.png">